mtags/src/main/scala/scala/meta/internal/mtags/ScalaToplevelMtags.scala

```diff
@@ -155,13 +162,29 @@ class ScalaToplevelMtags(
           emitMember(false, currRegion.owner)
           loop(indent, isAfterNewline = false, currRegion, newExpectTemplate)
         // also covers extension methods because of `def` inside
-        case DEF if dialect.allowExtensionMethods && currRegion.isExtension =>
+        case DEF
+            // extension group
+            if (dialect.allowExtensionMethods && currRegion.isExtension) =>
           acceptTrivia()
           val name = newIdentifier
           withOwner(currRegion.owner) {
             term(name.name, name.pos, Kind.OBJECT, 0)
           }
           loop(indent, isAfterNewline = false, region, newExpectIgnoreBody)
+        // inline extension method `extension (...) def foo = ...`
+        case DEF if expectTemplate.map(needToParseExtension).getOrElse(false) =>
+          expectTemplate match {
+            case None => fail("never happens")
+            case Some(expect) =>
+              val next =
+                expect.startIndentedRegion(currRegion, expect.isExtension)
```

Comment:
Why do we need to start indented region if the PR is about  inline extensions?

Reply:
Right, we didn't need it :+1: https://github.com/scalameta/metals/pull/4199/commits/5f699f0b1e5c65cf998a9f5fb8bde9c0237d89ba